x-pack/plugins/apm/server/lib/connections/get_connection_stats/get_destination_map.ts

```diff
@@ -69,58 +72,122 @@ export const getDestinationMap = ({
       offset,
     });
 
-    const response = await apmEventClient.search('get_exit_span_samples', {
-      apm: {
-        events: [ProcessorEvent.span],
-      },
-      body: {
-        size: 0,
-        query: {
-          bool: {
-            filter: [
-              { exists: { field: SPAN_DESTINATION_SERVICE_RESOURCE } },
-              ...rangeQuery(startWithOffset, endWithOffset),
-              ...filter,
-              ...excludeRumExitSpansQuery(),
-            ],
-          },
+    const response = await apmEventClient.search(
```

Comment:
This is setup for a quick POC, the queries will either be combined into one or refactored.